.github/workflows/ci.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
-      - name: Use Node.js LTS (16.x)
+        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
+      - name: Use Node.js LTS (20.x)
         uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af
         with:
-          node-version: '16'
+          node-version: '20'
```

Comment:
NodeJS 22 is LTS, so maybe you want to update to that?

Reply:
will try, sec. Still running 20 locally

Reply:
updated to 22